docs/cmdline-opts/data.md

```diff
@@ -44,6 +46,12 @@ is told to read from a file like that, carriage returns, newlines and null
 bytes are stripped out. If you do not want the @ character to have a special
 interpretation use --data-raw instead.
 
+If your filename ends with an exclamation mark followed by a numeric value
+<skip_bytes>, the first <skip_byes> bytes from the File are ignored. If you write two
+numeric values separated by a minus sign, this is considered a range from - to.
+For example, @notes.txt!1-3 uploads the second and third bytes of the file
+notes.txt.
```

Comment:
Don't you think it would maybe be worth making this range setup use the same logic as `--range` ? The numbers are the inclusive byte offsets:

## 0-499
specifies the first 500 bytes

## 500-999
specifies the second 500 bytes

## -500
specifies the last 500 bytes

## 9500-
specifies the bytes from offset 9500 and forward